docs/API.md

```diff
@@ -510,73 +510,85 @@ minioClient.setBucketVersioning('bucketname', versioningConfig, function (err) {
 
 <a name="setBucketReplication"></a>
 
-### setBucketReplication(bucketName, replicationConfig, callback)
+### setBucketReplication(bucketName, replicationConfig)
 
 Set replication config on a Bucket
 
 **Parameters**
 
-| Param               | Type       | Description                                      |
-| ------------------- | ---------- | ------------------------------------------------ |
-| `bucketName`        | _string_   | Name of the bucket.                              |
-| `replicationConfig` | _object_   | replicationConfig Configuration as a JSON Object |
-| `callback(err)`     | _function_ | Callback is called with `err` in case of error.  |
+| Param               | Type     | Description                                      |
+| ------------------- | -------- | ------------------------------------------------ |
+| `bucketName`        | _string_ | Name of the bucket.                              |
+| `replicationConfig` | _object_ | replicationConfig Configuration as a JSON Object |
 
 **Example**
 
-```js
-const arnFromMcCli =
-  'arn:minio:replication::1277fcbe7df0bab76ab0c64cf7c45a0d27e01917ee5f11e913f3478417833660:destination'
+Please refer to : [set-bucket-replication.mjs](..%2Fexamples%2Fset-bucket-replication.mjs)
```

Comment:
why percent-encoded?

Reply:
i will remove this. as it is redundant. (API.md) has reference already.